src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+let right;
+let left;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (click) => {
+    if (click.button === 0) {
+      setTimeout(() => {
+        resolve();
+      }, 3000);
+      left = true;
+    }
+  });
```

Comment:
the logic here is not matching the task. could you fix it please?
![image](https://user-images.githubusercontent.com/22824431/225311716-0e58f01a-3c69-4fbb-96ca-b8798bc9f689.png)